client/js/components/GameView/GameOverlays.vue

```diff
@@ -83,7 +83,22 @@ export default {
       waitingForOpponentToPickFromScrap: ({ game }) => game.waitingForOpponentToPickFromScrap,
       waitingForOpponentToPlayFromDeck: ({ game }) => game.waitingForOpponentToPlayFromDeck,
     }),
-    ...mapGetters(['isPlayersTurn', 'opponentQueenCount']),
+    ...mapGetters([
+      'isPlayersTurn',
+      'playerQueenCount',
+      'opponentQueenCount',
+      'opponent',
+      'hasGlassesEight',
+    ]),
+    waitingForOpponetToCounterMessage() {
+      const mayCounter = 'Opponent May Counter';
+      const mustResolve = 'Opponent Must Resolve';
+      const opponentHasTwo = this.opponent.hand.some((card) => card.rank === 2);
+      if (this.playerQueenCount || (this.hasGlassesEight && !opponentHasTwo)) {
+        return mustResolve;
+      }
+      return mayCounter;
```

Comment:
```suggestion
      return (this.playerQueenCount || (this.hasGlassesEight && !opponentHasTwo)) ?
        mustResolve : mayCounter;
```
Don't merge this as-is because prettier will yell, but you could also one line this. Probably cleaner with the if block though.

Reply:
Cool thanks. I was on the fence about a one-liner. It's helpful to see it written out like this. I think I'll stick with the conditional just to be explicit and obvious